app/views/foreman/job_templates/update_packages_by_search_query_-_katello_ansible_default.erb

```diff
@@ -0,0 +1,32 @@
+<%#
+kind: job_template
+name: Update packages by search query - Katello Ansible Default
+job_category: Katello
+description_format: 'Update package(s) %{Packages search query}'
+feature: katello_package_remove_by_search
+provider_type: Ansible
+template_inputs:
+- name: Packages search query
+  description: Filter criteria for packages to be updated.
+  input_type: user
+  required: false
+- name: Selected update versions
+  description: JSON string of selected package versions to be updated, in the format [ nvra ]. Leave blank to upgrade to latest available version.
+  input_type: user
+  required: false
+  value_type: plain
+%>
+<% package_names = @host.package_names_for_job_template(
+  action: 'update',
+  search: input('Packages search query'),
+  versions: input('Selected update versions')
+) -%>
+---
+- hosts: all
+  tasks:
+    - package:
+        name: 
+<% package_names.each do |package_name| -%>
+          - <%= package_name %>
+<% end -%>
+        state: present
```

Comment:
```suggestion
<%= render_template('Package Action - Ansible Default', :action => 'update', :package => package_names.join(' ')) %>
```
You have written `state: present`

Reply:
There is really an error here: `state: present` -> `state: latest` or use `Package Action - Ansible Default` with `latest`
I don't put the rendered template because my test machine is outdated (long list) but we have the same layout than the others.

Scenario tested:
- update one package by clicking and it works correctly.
- update two packages by clicking and it works correctly.
- update with search query * and I see the full list and it works correctly
- update with blank search query => the lists is [] so a correct yaml. rc=0 and "Nothing to do"
- update with a query to a uptodate package `name ~ acl` same than blank query. In the proposed file: it would cause to trigger a `yum update` because the list is empty.